tests/variable_creation_test.py

```diff
@@ -475,6 +475,35 @@ def test_arange():
     assert sc.identical(var, expected)
 
 
+def test_arange_datetime_from_int():
+    var = sc.arange('t', 14, 65, 21, unit='s', dtype='datetime64')
+    expected = sc.datetimes(dims=['t'], values=[14, 35, 56], unit='s')
+    assert sc.identical(var, expected)
+
+
+def test_arange_datetime_from_np_datetime64():
+    var = sc.arange('t', np.datetime64('2022-08-02T11:18'),
+                    np.datetime64('2022-08-02T11:52'), 15)
+    expected = sc.datetimes(
+        dims=['t'],
+        values=['2022-08-02T11:18', '2022-08-02T11:33', '2022-08-02T11:48'],
+        unit='min')
+    assert sc.identical(var, expected)
+
+
+def test_arange_datetime_from_str():
```

Comment:
Missing a test for `arange_datetime_from_scipp_datetime`?

Reply:
Done